wallet_txn_doublespend.py failed (flaky test: timeout)

Some tests in `wallet_txn_doublespend.py` fail randomly due to timeouts.

Here the first part of the log:
```
wallet_txn_doublespend.py failed, Duration: 63 s
stdout:
2019-01-29 13:31:57.798000 TestFramework (INFO): Initializing test directory /tmp/unite_test_runner_20190129_132150/wallet_txn_doublespend_565
2019-01-29 13:31:57.799000 TestFramework (INFO): Debug file at /tmp/unite_test_runner_20190129_132150/wallet_txn_doublespend_565/node0/regtest/debug.log
Starting node 0 with args: 
Starting node 1 with args: 
Starting node 2 with args: 
Starting node 3 with args: 
2019-01-29 13:33:00.804000 TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/home/travis/build/dtr-org/unit-e/build/unite-x86_64-unknown-linux-gnu/test/functional/test_framework/test_framework.py", line 125, in main
    self.run_test()
  File "/home/travis/build/dtr-org/unit-e/build/unite-x86_64-unknown-linux-gnu/test/functional/wallet_txn_doublespend.py", line 107, in run_test
    sync_blocks(self.nodes)
  File "/home/travis/build/dtr-org/unit-e/build/unite-x86_64-unknown-linux-gnu/test/functional/test_framework/util.py", line 385, in sync_blocks
    maxheight, "".join("\n  {!r}".format(tip) for tip in tips)))
AssertionError: Block sync to height 202 timed out:
  {'height': 200, 'hash': '2137a08877c1ad58f0ce4c60987b9042ef78fff88ad5a57c0ee994e60c37ae06'}
  {'height': 200, 'hash': '2137a08877c1ad58f0ce4c60987b9042ef78fff88ad5a57c0ee994e60c37ae06'}
  {'height': 202, 'hash': '3a2cb89467b999222bfe89611b0c981aa2ceaa9f4521a7d5a50f03ec7c8a8384'}
  {'height': 202, 'hash': '3a2cb89467b999222bfe89611b0c981aa2ceaa9f4521a7d5a50f03ec7c8a8384'}
2019-01-29 13:33:00.805000 TestFramework (INFO): Stopping nodes
2019-01-29 13:33:01.063000 TestFramework (WARNING): Not cleaning up dir /tmp/unite_test_runner_20190129_132150/wallet_txn_doublespend_565
2019-01-29 13:33:01.063000 TestFramework (ERROR): Test failed. Test logging available at /tmp/unite_test_runner_20190129_132150/wallet_txn_doublespend_565/test_framework.log
2019-01-29 13:33:01.064000 TestFramework (ERROR): Hint: Call /home/travis/build/dtr-org/unit-e/build/unite-x86_64-unknown-linux-gnu/test/functional/combine_logs.py '/tmp/unite_test_runner_20190129_132150/wallet_txn_doublespend_565' to consolidate all logs
```

--- a/test/functional/wallet_txn_doublespend.py
+++ b/test/functional/wallet_txn_doublespend.py
@@ -25,6 +25,7 @@ def create_and_sign_tx(node, inputs, outputs):
 class TxnMallTest(UnitETestFramework):
     def set_test_params(self):
         self.num_nodes = 4
+        self.extra_args = [['-maxtipage=1000000000']] * self.num_nodes
 
     def add_options(self, parser):
         parser.add_option("--mineblock", dest="mine_block", default=False, action="store_true",

